db/schema.rb

```diff
@@ -36,14 +36,14 @@
   create_table "pieces", force: :cascade do |t|
     t.string "type"
     t.integer "game_id"
-    t.integer "white_player_id"
-    t.integer "black_player_id"
     t.integer "rank"
     t.string "file"
     t.integer "move_id"
     t.boolean "is_captured", default: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.bigint "user_id"
+    t.index ["user_id"], name: "index_pieces_on_user_id"
```

Comment:
Hi Gary, I'm learning about foreign keys now to be able to merge your PR! Thank you for the links on Slack.
One question:  Were the lines 45, 46, and 66 added by Rails when you ran commands to create the migration? Or did you select the data type `bigint` (never came across it before) etc?

Reply:
No problem Claire.  Yes, those were automatically added by Rails when I ran the migration.  I know in the course, I believe we manually wrote down <field_name>_id and set up the relationship in the model.  There is another way to explicitly set a foreign key that I learned and that is to do the following:

rails generate migration AddUserToPieces user:references

This will automatically create a user_id field that references the Users table and from the looks of it also adds an index as well which I didn't even know about.

Reply:
In regards to the bigint data type you can check out http://www.postgresqltutorial.com/postgresql-integer/  , I think Rails selects this to assume that the number of rows could possibly exceed the max value of an integer data type

Reply:
Thank you for the info!
I've seen your [more recent Pull Request](https://github.com/rosalinekarr/WarAndPiece/pull/15) with changes to schema.rb. I had not noticed it when looking at this branch because I'm not assignee/reviewer. I'm going to go ahead and merge both so we can move forward. Hope it's ok!